### ISSUE TITLE
Broken notebook cta_sensitivity.ipynb

### ISSUE THREAD

**Issue by Bultako** [gammapy/gammapy#1943]
This noteboook is not delivered by `gammapy download tutorials` but we keep in this repo because it is referenced from other pages in the documentation.

```bash
$ gammapy jupyter --src=./tutorials/cta_sensitivity.ipynb test
INFO:gammapy.scripts.jupyter:   ... EXECUTING tutorials/cta_sensitivity.ipynb
INFO:gammapy.scripts.jupyter:   ... Executing duration: 0.07 mn
INFO:gammapy.scripts.jupyter:   ... FAILED
INFO:gammapy.scripts.jupyter:

ImportError in cell [2]

--SOURCE CODE: 
from gammapy.irf import CTAPerf
from gammapy.spectrum import SensitivityEstimator

--TRACEBACK: 
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-b9f53885d5c3> in <module>
----> 1 from gammapy.irf import CTAPerf
      2 from gammapy.spectrum import SensitivityEstimator

ImportError: cannot import name 'CTAPerf' from 'gammapy.irf' (/Users/jer/git/gammapy/gammapy/gammapy/irf/__init__.py)
```

As an add-on to this issue, when I tried to fix the notebook, I found that spectrum.SensitivityEstimator breaks with the DC1 IRFs. It works fine with those in https://github.com/gammapy/gammapy-extra/tree/master/datasets/cta/perf_prod2/point_like_non_smoothed

**Comment by cdeil** [comment]
@Bultako - Please mark the notebook as broken for now.

This is known, @registerrier said he would fix this.
Related issues (mostly all about the same task): #1824, #1551, #876, #809 

@registerrier - Please make this your priority in Gammapy, and please do it in steps to keep the basic functionality of SpectrumSimulation and SensitivityEstimator working, so that it's fixed for v0.10, even if developing complete and good new background model and dataset classes will take many months.

**Comment by registerrier** [comment]
This issue should be solved after PR #2000.
